data/test_response_selector_bot/config.yml

```diff
@@ -5,10 +5,10 @@ pipeline:
   - name: "CountVectorsFeaturizer"
   - name: "DIETClassifier"
     entity_recognition: False
-    epochs: 5
+    epochs: 50
```

Comment:
Any reason we need `50`? it will make the tests a lot slower...

Reply:
The less training data you have, the more epochs you have to train for. At 5 you can find random seeds where the tests fail - even on main (and so they may fail if we just change the architecture slightly). We might get away with fewer epochs and a higher learning rate, but tests like tests/core/test_evaluation.py::test_retrieval_intent can fail randomly.

Reply:
Then can we not set it to a random seed that works and keep the epochs low?

Reply:
Ok, I've reduced the number of epochs and increased the learning rate for now.

Reply:
>Then can we not set it to a random seed that works and keep the epochs low?
Sorry, GitHub didn't show this until I refreshed the page. We can do this, but I would call it bad practice. If you change anything in the model, this would be like changing the random seed, and your test may suddenly fail for no reason.

Reply:
I think with the increased learning rate we might be fine. But success is never 100% guaranteed.

Reply:
Yeah, the point of unit tests is to test the code around the model rather than the effectiveness or accuracy of the model - so the most important thing is just having stability. And we need our unit tests to be fast so it doesn't hinder development and the CI .
However I do see your point about it being liable to change easily meaning having to update the test often which is not ideal. The other alternative is to mock the model, however I like this even less as it brings us further for reality. 
I would say the best (however not perfect) option is setting the random seed and having relatively low epochs.

Reply:
Ok, then let's keep it as it is now. From an ML perspective it makes sense to have a higher learning rate here, and the test succeeds.